sentry/sentry.conf.example.py

```diff
@@ -363,3 +363,9 @@ def get_internal_network():
 # https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-CSRF_TRUSTED_ORIGINS
 
 # CSRF_TRUSTED_ORIGINS = ["https://example.com", "http://127.0.0.1:9000"]
+
+#################
+# JS SDK Loader #
+#################
+
+JS_SDK_LOADER_DEFAULT_SDK_URL = "https://browser.sentry-cdn.com/%s/bundle%s.min.js"
```

Comment:
The config is from here: https://github.com/getsentry/sentry/blob/b2a1b6119b1bcb6f8d4231be9aa1b214867a04a0/src/sentry/conf/server.py#L2797-L2812